website/docs/docs/cloud/manage-access/invite-users.md

```diff
@@ -0,0 +1,57 @@
+id: "invite-users"
+sidebar: "Invite users"
+---
+
+dbt Cloud makes it easy to invite new users to your environment out of the box. This feature is available to all dbt Cloud customers on Teams or Enteprise plans (Developer plans are limited to a single user).
+
+## Invite new users
+
+You must have proper permissions to invite new users:
+
+- [**Teams accounts**](/docs/cloud/manage-access/self-service-permissions) &mdash; must have `member` or `owner` permissions.
+- [**Enterprise accounts**](/docs/cloud/manage-access/enterprise-permissions) &mdash; must have `admin`, `account admin`, `project creator`, or `security admin` permissions.
+
+To invite users: 
+
+1. Open the **Account Settings** from the gear menu in your dbt Cloud account.
+2. From the left sidebar, select **Users**.
+
+<Lightbox src="/img/docs/dbt-cloud/access-control/users-screen.png" title="The user screen in settings"/>
+
+3. Click on **Invite Users**.
+
+<Lightbox src="/img/docs/dbt-cloud/access-control/invite-users.png" title="The invite users pane"/>
+
+4. In the **Email Addresses** field, enter the email addresses of the users you would like to invite separated by comma, semicolon, or a new line.
+5. Select the license type for the batch of users from the **License** dropdown.
+6. Select the group(s) you would like the invitees to belong to.
+7. Click **Send Invitations**.
+    - If the list of invitees exceeds the number of licenses your account has availabe, you will receive a warning when you click **Send Invitations** and the invitations will not be sent. 
+
+
+## User experience
+
+dbt generates and sends the email to the addresses defined in dbt Cloud. Be sure to allow traffic from `support@getdbt.com` to prevent emails from going to spam or being blocked (this is the originating email address for all [instances worldwide](/docs/cloud/about-cloud/regions-ip-addresses)).
+
+The email contains a link to create an account. When the user clicks on this they will be brought to one of two screens depending on whether SSO is configured or not.
+
+<Lightbox src="/img/docs/dbt-cloud/access-control/email-invite.png" title="Example or an email invitation"/>
+
+The default settings send the email, the user clicks the link, and is prompted to create their account:
+
+<Lightbox src="/img/docs/dbt-cloud/access-control/default-user-invite.png" title="Default user invitation"/>
+
+If SSO is configured for the environment, the user clicks the link, is brought to a confirmation screen, and presented with a link to authenticate against the company's identity provider:
+
+<Lightbox src="/img/docs/dbt-cloud/access-control/sso-user-invite.png" title="User invitation with SSO configured"/>
+
+Once the user completes this process, their email and user information will populate in the **Users** screen in dbt Cloud.
+
+## FAQ
+
+* Is there a limit to the number of users I can invite? _Your ability to invite users is limited to the number of licenses you have available._
```

Comment:
might be good to add something about if the email has a typo? what do they do if the email they invited is wrong? who do they reach out to?

Reply:
Do they need to reach out to someone? Wouldn't they send another invite? Is there something support needs to do?

Reply:
i dont think they reach out to support (although i def do remember users reaching out to support when they had typos,etc). but i believe they would need to create an account with the correct email. @BrJan can confirm though! [also this may have insight](https://www.notion.so/dbtlabs/How-to-assist-a-customer-that-registered-with-the-incorrect-email-address-8f08017f58d84983841c0df5dfc25dd6?pvs=4)